osu.Game/Rulesets/UI/Playfield.cs

```diff
@@ -37,8 +40,10 @@ protected Playfield(float? customWidth = null, float? customHeight = null)
         }
 
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(OsuConfigManager config)
         {
+            ShowPerfectJudgements = config.Get<bool>(OsuSetting.ShowPerfectJudgements);
```

Comment:
this would require the playfield to be reloaded for changed settings to take effect. you could get and use the bindable setting from config for live updates (for example, changing the setting while watching a replay)